bin/run-wp-unit-tests.sh

```diff
@@ -21,16 +21,6 @@ fi
 
 npm run build || exit 1
 
-# Make sure phpegjs parser is up to date
-node bin/create-php-parser.js || exit 1
-if ! git diff --quiet --exit-code lib/parser.php; then
```

Comment:
This check is already accounted for in CI by:

- The files are automatically built as a `postinstall` step
- CI runs [`check-local-changes`](https://github.com/WordPress/gutenberg/blob/ab0acc399ea3457833b2f7f50fd9a3bff90a57d0/package.json#L155) to verify no commit-able files

Reply:
👍